go/src/lib.rs

```diff
@@ -379,6 +398,36 @@ pub unsafe extern "C" fn command(
                 command_response.float_value = num;
                 Ok(Some(command_response))
             }
+            Value::Boolean(boolean) => {
+                command_response.bool_value = boolean;
+                Ok(Some(command_response))
+            }
+            Value::Array(array) => {
+                let mut vec_elements_len: Vec<c_long> = Vec::new();
```

Comment:
if we know the length in advance, use `Vec::with_capacity` to reduce memory allocations

Reply:
Done